apps/src/dance/Dance.js

```diff
@@ -176,16 +179,26 @@ Dance.prototype.init = function (config) {
 
   ReactDOM.render(
     <Provider store={getStore()}>
-      <AppView
-        visualizationColumn={
-          <DanceVisualizationColumn
-            showFinishButton={showFinishButton}
-            setSong={this.setSongCallback.bind(this)}
-            resetProgram={this.reset.bind(this)}
-          />
+      <ErrorBoundary
+        fallback={<ErrorFallbackPage />}
```

Comment:
Can we make a note (here or elsewhere) that we're using the Lab2 ErrorFallbackPage for now, but should probably refactor after HOC to reduce coupling with Lab2 code?

Reply:
@sanchitmalhotra126 - just curious. Are you saying that we should create a Dance legacy version of `ErrorBoundary` after HoC?

Reply:
`ErrorBoundary` is generic and can be shared, but `ErrorFallbackPage` uses some Lab2 styling in its CSS, so we'd just want to double check to see if it's okay to keep that shared, or if we want to abstract out some of the styling.

Reply:
Thanks! Just added note to [Dance AI Tech Debt doc](https://docs.google.com/document/d/1PxF26am8yamMszcEUptrfVDwl6rgZzSEr8pzuqjlu_A/edit).